yarn.lock

```diff
@@ -2283,6 +2283,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@opentelemetry/context-async-hooks@npm:^1.17.1":
```

Comment:
I would not expect to see two copies of this installed; `@opentelemetry/*` packages specifically tend to rely on shared singletons/state/identity. Unfortunately `yarn dedupe` won't help here, you might have to manually edit `yarn.lock` to get both semver ranges to resolve to `1.17.1`.